committeeoversightapp/templates/hearing_detail.html

```diff
@@ -0,0 +1,116 @@
+{% extends 'base.html' %}
+{% load committeeoversight_tags %}
+
+{% block content %}
+<div class="row justify-content-center">
+  <div class="col col-lg-11">
+  <div class="hearing-form card">
+    <h2 class="text-center">{{hearing.name|title }}</h2>
+    <br />
+
+    <table class="table table-striped table-bordered" id="hearing-table">
+     <thead>
+     </thead>
+     <tbody>
+       <tr>
+         <td><b>Hearing Date</b></td>
+         <td>{{hearing.start_date}}</td>
```

Comment:
Since `start_date` is stored as a raw string, the output isn't very friendly:

<img width="801" alt="Screen Shot 2019-11-06 at 5 08 27 PM" src="https://user-images.githubusercontent.com/14170650/68345857-1389c580-00b8-11ea-9b0b-bcae4dc7a759.png">

Not a required change I don't think, but if it bothers you then you could think about doing some parsing to try to format the date.

Reply:
This formatting is site-wide and I'm not sure what Lugar's preferences are, so I'd prefer to handle it outside of this issue!

Reply:
Sounds reasonable to me 👍